libraries/Microsoft.Bot.Builder.Dialogs.Adaptive.Testing/Mocks/MockSkillClaimMiddleware.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Security.Claims;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder.Skills;
+using Microsoft.Bot.Connector.Authentication;
+
+namespace Microsoft.Bot.Builder.Dialogs.Adaptive.Testing.Mocks
+{
+    /// <summary>
+    /// Middleware for mocking skill claim.
+    /// </summary>
+    public class MockSkillClaimMiddleware : IMiddleware
+    {
+        private readonly ClaimsIdentity claimsIdentity;
+        private readonly SkillConversationReference skillConversationReference;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="MockSkillClaimMiddleware"/> class.
+        /// </summary>
+        /// <param name="mockCase">The mock case.</param>
+        /// <param name="parentBotId">Parent bot Id. If null, new GUID.</param>
+        /// <param name="skillBotId">Skill bot Id. If null, new GUID.</param>
+        public MockSkillClaimMiddleware(MockCase mockCase, string parentBotId = null, string skillBotId = null)
```

Comment:
>MockCase mockCase, string parentBotId = null, string skillBotId = null [](start = 40, length = 70)

Given the way you have this setup in TestScript.cs, how would you ever get any of these other cases or parameters?